content/information-distribution/node.md

```diff
@@ -31,12 +31,15 @@ messages timeline and [request the missing message from
 `C`](https://matrix.org/docs/spec/server_server/unstable.html#post-matrix-federation-v1-get-missing-events-roomid).
 
 In order to avoid a node being left out of a federation, nodes **must** send out
-a small message, named "ping" (2️⃣), to the Matrix room every twelve hours. This
+a small message, named "ping", to the Matrix room every twelve hours. This
 way, a node that's cut from most of a federation except for a few nodes will end
 up getting all of the information it missed. If a node is offline when it is
 supposed to send out a ping it **should** send it as soon as it gets back
 online.
```

Comment:
I would have added:
```markdown
Internally, sending pings triggers the [Matrix's `get_missing_events` federation
endpoint](https://matrix.org/docs/spec/server_server/unstable.html#post-matrix-federation-v1-get-missing-events-roomid)
and forces the Matrix server to query known federated servers to retrieve missing data.
```

Reply:
Actually, the reference to that endpoint is already mentioned in the page line 31, I didn't notice it. Therefore, I'm not adding that.